packages/manager-components/src/components/table/table.module.scss

```diff
@@ -0,0 +1,27 @@
+.table {
+  th {
+    text-align: center;
+    padding: 1rem 1rem;
+    border-bottom: solid 1px var(--ods-color-blue-200);
+  }
+
+  tbody {
+    td {
+      text-align: center;
+      padding: 1rem 1rem;
+      border-bottom: solid 1px var(--ods-color-blue-200);
+    }
+
+    tr:last-child {
+      border-bottom: solid 1px var(--ods-color-blue-200);
+    }
```

Comment:
**nitpick:** Isn't this redundant because we have `border-bottom` from `td`

Reply:
Yes, we can delete it